packages/components/src/components/popover/popover.jsx

```diff
@@ -3,16 +3,15 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import TinyPopover, { ArrowContainer } from 'react-tiny-popover';
 import Icon from '../icon';
+import { useHover } from '../../hooks/use-hover';
 
 const Popover = ({ ...props }) => {
     const ref = React.useRef();
 
-    const [is_popover_open, setIsPopoverOpen] = React.useState(false);
     const [is_bubble_open, setIsBubbleOpen] = React.useState(false);
     const [popover_ref, setPopoverRef] = React.useState(undefined);
     const bubble_ref = React.useRef(is_bubble_open);
-
-    const has_external_open_state = props.is_open !== undefined;
+    const [hoverRef, isHovered] = useHover();
```

Comment:
Could we rename to `hover_ref`?

Reply:
Right, I renamed both of these variables.